### DISCUSSION TITLE
2 plotters or more ram for temp. which is fastest

### DISCUSSION THREAD

**Discussion by svennand** [madMAx43v3r/chia-plotter#906] [General]
Im wondering if i should have two plotters to plot. Or if i should add more ram to my one server to have ram and temp in mem vs ssd temp. Anyone compared the speed difference?

I have one server with xeon 5675 cpu 196gb ram ddr3 and 2x 1TB ssd in raid 0 for temp.
I now have another server with xeon 2667, no ram and no drive. Should i buy ram/drive for this or just move disk/ram over to xeon 2667 and buy more memory.

With the first server i now make a plot every 55min.

**Comment by hajes** [comment]
AMD Ryzen 3950X using 15/30 cores ~35min. two plots at once ~ 77 min. CPU bottleneck.

If you have 32-128 cores CPU, RAM disk is your friend. Otherwise, CPU is usually your bottleneck. With RAM disk you reduce SSD wear only
